### PR TITLE
Feature/tda 28

### PR DESCRIPTION
**Pull request by duprijil**

### CODE REVIEW

File: dash/pages/poc.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+@callback(
+    Output('q15_graph', 'figure'),
+    State('q15_graph', 'figure'),
+    State('q15_daterange', 'start_date'),
+    State('q15_daterange', 'end_date'),
+    Input('q15_button', 'n_clicks'),
+    prevent_initial_call=True,
+)
+def figure_q15(figure: dict = None, start_date: str = START_DATE.strftime(QUERY_TIME_FORMAT),
```
Comment (costezki) [inline dash/pages/poc.py:48]: in the MVP we want a better way of organising these aspects.

Comment (costezki) [reply dash/pages/poc.py:48]: Perhaps are fine for now in the POC.

File: infra/dash/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FROM python:3.8.10-slim-buster
```
Comment (costezki) [inline infra/dash/Dockerfile:1]: this should be reviewed by @Dragos0000

File: infra/dash/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+version: '3.8'
```
Comment (costezki) [inline infra/dash/docker-compose.yml:1]: to be revised by @Dragos0000

File: infra/traefik/letsencrypt/acme.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (costezki) [inline infra/traefik/letsencrypt/acme.json:1]: Please remove this file from GIT. We cannot have any security info in GIT.
It should be added into gitignore.

File: requirements.txt [hunk 5]
```diff
@@ -1,2 +1,4 @@
-https://github.com/meaningfy-ws/ted-sws/archive/main.zip
-pymongo~=4.0.2
\ No newline at end of file
+git+https://github.com/OP-TED/ted-rdf-conversion-pipeline.git@1.0.0-beta
```
Comment (costezki) [inline requirements.txt:1]: @Dragos0000 @CaptainOfHacks 
We could consider creating a new tag that integrates the latest improvements.

File: resources/sparql_queries/q1.rq [hunk 6]
```diff
@@ -1,32 +0,0 @@
-PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
```
Comment (costezki) [inline resources/sparql_queries/q1.rq:1]: Why is this file deleted? Where are the queries now?

**Review by costezki — DISMISSED** [review]
Better to have this reviewed by colleagues.

File: dash/pages/poc.py (hunk 1 above)
Comment (duprijil) [reply dash/pages/poc.py:48]: Yes. This is for POC, to have a pipeline from implementation to result on our page. Further, it remains to be arranged correctly

File: infra/dash/docker-compose.yml (hunk 3 above)
Comment (duprijil) [reply infra/dash/docker-compose.yml:1]: compose version was chosen according to the versions used in ted sws

File: infra/dash/Dockerfile (hunk 2 above)
Comment (duprijil) [reply infra/dash/Dockerfile:1]: the given image was chosen as there is no special official image for the dash

File: infra/traefik/letsencrypt/acme.json (hunk 4 above)
Comment (duprijil) [reply infra/traefik/letsencrypt/acme.json]: done

File: resources/sparql_queries/q1.rq (hunk 6 above)
Comment (duprijil) [reply resources/sparql_queries/q1.rq:1]: For now, the resources are in dags/resources/sparql_queries. They will be put back in the main folder, after some changes in DAGs. For this, it is necessary to discuss the TDA architecture.

File: Makefile [hunk 7]
```diff
@@ -183,6 +201,8 @@ init-saxon:
 	@ cd .saxon && unzip SaxonHE10-6J.zip && rm -rf SaxonHE10-6J.zip
 
 
+start-project-services: | create-env-airflow start-airflow start-allegro-graph start-fuseki start-minio
```
Comment (CaptainOfHacks) [inline Makefile:204]: remove this

File: Makefile [hunk 8]
```diff
@@ -183,6 +201,8 @@ init-saxon:
...
+stop-project-services: | stop-airflow stop-allegro-graph stop-fuseki stop-minio
```
Comment (CaptainOfHacks) [inline Makefile:205]: remove this

File: Makefile [hunk 9]
```diff
@@ -265,6 +285,11 @@ prod-dotenv-file: guard-VAULT_ADDR guard-VAULT_TOKEN vault-installed
 	@ echo XML_PROCESSOR_PATH=${XML_PROCESSOR_PATH} >> .env
 	@ echo AIRFLOW_INFRA_FOLDER=~/airflow-infra/ted-data-prod >> .env
 	@ echo AIRFLOW_WORKER_HOSTNAME=${HOSTNAME} >> .env
+	@ vault kv get -format="json" ted-prod/airflow | jq -r ".data.data | keys[] as \$$k | \"\(\$$k)=\(.[\$$k])\"" >> .env
```
Comment (CaptainOfHacks) [inline Makefile:288]: remove this

File: Makefile [hunk 10]
```diff
@@ -265,6 +285,11 @@ prod-dotenv-file: guard-VAULT_ADDR guard-VAULT_TOKEN vault-installed
...
+	@ vault kv get -format="json" ted-prod/minio | jq -r ".data.data | keys[] as \$$k | \"\(\$$k)=\(.[\$$k])\"" >> .env
```
Comment (CaptainOfHacks) [inline Makefile:289]: remove this

File: Makefile [hunk 11]
```diff
@@ -265,6 +285,11 @@ prod-dotenv-file: guard-VAULT_ADDR guard-VAULT_TOKEN vault-installed
...
+	@ vault kv get -format="json" ted-prod/mongo-db | jq -r ".data.data | keys[] as \$$k | \"\(\$$k)=\(.[\$$k])\"" >> .env
```
Comment (CaptainOfHacks) [inline Makefile:290]: remove this

File: Makefile [hunk 12]
```diff
@@ -265,6 +285,11 @@ prod-dotenv-file: guard-VAULT_ADDR guard-VAULT_TOKEN vault-installed
...
+	@ vault kv get -format="json" ted-prod/fuseki | jq -r ".data.data | keys[] as \$$k | \"\(\$$k)=\(.[\$$k])\"" >> .env
```
Comment (CaptainOfHacks) [inline Makefile:291]: remove this

File: Makefile [hunk 13]
```diff
@@ -265,6 +285,11 @@ prod-dotenv-file: guard-VAULT_ADDR guard-VAULT_TOKEN vault-installed
...
+	@ vault kv get -format="json" ted-prod/agraph | jq -r ".data.data | keys[] as \$$k | \"\(\$$k)=\(.[\$$k])\"" >> .env
```
Comment (CaptainOfHacks) [inline Makefile:292]: remove this

**Review by CaptainOfHacks — APPROVED** [review]

**Pull request merged**